share/dkg/dkg.go

```diff
@@ -0,0 +1,654 @@
+// Package dkg implements the protocol described in
+// "Secure Distributed Key Generation for Discrete-Log
+// Based Cryptosystems".
```

Comment:
What about http://groups.csail.mit.edu/cis/pubs/stasio/vss.ps.gz - is that an official link?